src/pages/SignIn/SignIn.tsx

```diff
@@ -99,6 +99,21 @@ const SignIn: React.FC = () => {
         sessionStorage.removeItem("NOT_WHITELISTED_IP_MESSAGE");
         return;
       }
+      case "MARK_FOR_DELETION_ERROR": {
```

Comment:
Let's make this generic. There may be other cases for Unauthorized as well.

```suggestion
      case "unauth": {
```

Reply:
so under `unauth` ErrorCode, the session storage key would be something related to mark for deletion or that would be generic as well like `unauth`

Reply:
It will also be generic, because we don't have a separate differentiator for MARK FOR DELETION.